### DISCUSSION TITLE
Missing data

### DISCUSSION THREAD

**Discussion by Dabs212** [JerBouma/FinanceDatabase#23] [Q&A] [answered]
![No_ETFs](https://user-images.githubusercontent.com/75166223/194714823-5d5571d8-c83b-4f72-8d90-e7f6da91d22a.JPG)

There are No ETFs for following Sectors!

**Comment by JerBouma** [comment, marked as answer]
This should now be corrected. If a sector still has no symbols, make sure to set `exclude_exchanges` to False.
